src/pymovements/dataset/dataset_files.py

```diff
@@ -575,11 +564,16 @@ def add_fileinfo(
     )
 
     # Cast columns from fileinfo according to specification.
-    _schema_overrides = definition.filename_format_schema_overrides['gaze']
-    df = df.with_columns([
-        pl.col(fileinfo_key).cast(fileinfo_dtype)
-        for fileinfo_key, fileinfo_dtype in _schema_overrides.items()
-    ])
+    resource_definitions = definition.resources.filter('gaze')
```

Comment:
this method is only used by `load_event_files()` which expects the events filename pattern to be the same as for `gaze` files (as we currently expect them to be generated with pymovements). this is a bit confusing but we will improve on that later on when merging `precomputed_events` content into `events`, but there's still some work to do until then.